### ISSUE TITLE
Validation set error is not shown

### ISSUE THREAD

**Issue by ab93** [numaproj/numalogic#150]
**Describe the bug**
Validation error is not shown if provided a validation dataset in the autoencoder trainer.

**To Reproduce**
Provide a validation dataloader to the autoencoder trainer

**Expected behavior**
Validation scores should show up in addition to the training scores.

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We often sort issues this way to know what to prioritize.
